config/environments/development.rb

```diff
@@ -38,19 +38,19 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.delivery_method = :sendmail
-config.action_mailer.perform_deliveries = true
-config.action_mailer.raise_delivery_errors = true
-config.action_mailer.default_options = {from: 'nicolasolaya22@gmail.com'}
-config.action_mailer.delivery_method = :smtp
-config.action_mailer.smtp_settings = {
-address:              'smtp.gmail.com',
-port:                 587,
-domain:               'gmail.com',
-user_name:            Rails.application.credentials.gmail[:email],
-password:            Rails.application.credentials.gmail[:password],
-authentication:       'plain',
-enable_starttls_auto: true  }
-config.action_mailer.default_url_options = {:host =>'localhost:3000'}
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_options = {from: 'nicolasolaya22@gmail.com'}
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  address:              'smtp.gmail.com',
+  port:                 587,
+  domain:               'gmail.com',
+  user_name:            Rails.application.credentials.gmail[:email],
```

Comment:
- [ ] Good job trying to update your `confirmable` feature to the `User authentication` as mentioned by the previous reviewer. However, I noticed that your application is still having issue to run properly. Right now, when I try to sign up with a new User, I receive the below error. I think the the issue can come from the above configuration as it was blocking me to create a new database unless I commented them out.
![Screenshot from 2023-06-21 04-30-31](https://github.com/NicolasAndrehh/Rails-Blog-App/assets/83042742/3ae0f2bb-7fd9-4afd-a42e-8b13bf6fbb55)

Reply:
Hey @tresorsawasawa, thanks for the feedback, I have a question, is this the error you get when trying to start the server? Or were you able to start the server and then you got that error please give me more details 🙏.
I suspect that is because I didn't commit the credentials.env because if I commit it my email would be vulnerable
![image](https://github.com/NicolasAndrehh/Rails-Blog-App/assets/111377816/857353d1-20e0-466c-859c-e27412ded890)

Reply:
Hey @NicolasAndrehh 

The issue occurred when I wanted to create a new database showing the selected code to be the origin.

Then I commented those code lines, so I was able to create the database and run the server.

But after filling the form to sign up with a new user, I got that error when I tried to submit the form by clicking the "Signup" button